modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the Image Registry Operator for use with Azure.
+
+.Procedure 
+
+. Enter the following command to obtain the VNet and subnet names that are used in the cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -nopenshift-machine-api -ojson|jq '.items[1].spec.template.spec.providerSpec.value | (.vnet, .subnet)'
+----
+
+. Edit the Image Registry Operator `config/instance` object and configure the private endpoint using your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+          internal:
+            subnetName: <subnet_name>
+            vnetName: <vnet_name>
+# ...
+----
+
+. Wait until the Image Registry Operator configures the private endpoint. Enter the following command to return the endpoint name:
++
+[source,terminal]
+----
+$ oc get configs.imageregistry/cluster -o=jsonpath="{.spec.storage.azure.privateEndpointName}" -w
+----
+
+. Enter the following command to enable the default route and disable `redirect` on the Image Operator configuration:
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"defaultRoute": true, "disableRedirect": true}}'
```

Comment:
Enabling the default route is not a required step to configure private endpoints.
Disabling redirect is only required when users already expose the registry (via defaultRoute or a custom route).

Enabling the route (and disabling redirect) should be located in the verification steps. 
```suggestion
. (Optional) If the registry is exposed via a route (defaultRoute or otherwise), disable `redirect` on the Image Operator configuration:
+
[source,terminal]
----
$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"disableRedirect": true}}'
```

Reply:
Fixed language in both procedures.